src/Stripe.net/Entities/Sources/SourceAuBecsDebit.cs

```diff
@@ -13,7 +14,11 @@ public class SourceAuBecsDebit : StripeEntity
         [JsonProperty("fingerprint")]
         public string Fingerprint { get; set; }
 
+        [Obsolete("This property is deprecated, please use Last4 going forward.")]
```

Comment:
I realized that `AccountNumber` is in the resource and shouldn't be right? It's not returned on the resource. We likely should also mark it as obsolete.

Reply:
@remi-stripe  you are right. AccountNumber is not returned any more. Make it as obsolete too.